worker_isolate.dart

```diff
@@ -15,7 +15,9 @@ import 'package:sintr_common/bucket_utils.dart';
 import 'package:sintr_common/configuration.dart' as config;
 import 'package:sintr_common/logging_utils.dart' as log;
 
-import 'package:smart/completion_model/analyse_path.dart';
+import 'package:smart/completion_model/analyse_path.dart' as completion_model;
+import 'package:smart/discovery_model/analyse_path.dart' as discovery_model;
+
```

Comment:
I'd love some comments on this file :) To help a poor reviewer looking at the code for the first time.